### ISSUE TITLE
Is it ready or just for testing?

### ISSUE THREAD

**Issue by ugintl** [AcroMedia/drupal-orange-project#17]
Is it ready to be used to create a site for a client?

**Comment by 7o7o** [comment]
I've been using it for some months and i find it very useful. But as everything with Drupal it comes with some complexity so be aware that you could face different complications in the future...
In any case it will help you learn many things that can be done with Drupal E-commerce. Good luck.

**Comment by ugintl** [comment]
I still tried to install it and sellected "ecommerce profile " during installation. When installation finished, I saw an unstyled page. Did it happpen to you?

**Comment by 7o7o** [comment]
Yes. There used to be the style.css file before but now you need to generate it yourself. To do that you have to install npm and gulp:

npm install: https://www.npmjs.com/get-npm
After npm is installed you can do `npm install` on the theme folder.

Then install gulp:
`npm install --global gulp-cli`

And then execute `gulp` on the orange-ecommerce theme. If you want to make modifications you should copy that theme and create a new one. You have more info in the README file of the theme.

After you execute gulp you should see a style.css file on the CSS folder. What gulp does is to grab all .scss files and mix in style.css. Your modifications should go on those .scss files, not on the style.css because it could be overwritten when you compile your scss files.

Maybe for people that is not front end developers it could be a good idea to include again the style.css files so the site don't look unstyled for the first time.
